OptimizelySDKCore/OptimizelySDKCore/Optimizely.h

```diff
@@ -201,26 +200,6 @@ extern NSString * _Nonnull const OptimizelyNotificationsUserDictionaryExperiment
 - (void)track:(nonnull NSString *)eventKey
        userId:(nonnull NSString *)userId;
 
-/** @deprecated. Use `track:userId:eventValue`.
- * Track an event
- * @param eventKey The event name
- * @param userId The user ID associated with the event to track
- * @param eventValue The event value (e.g., revenue amount)
- */
-- (void)track:(nonnull NSString *)eventKey
-       userId:(nonnull NSString *)userId
-   eventValue:(nonnull NSNumber *)eventValue __attribute((deprecated("eventValue is deprecated in track call. Use eventTags to pass in revenue value instead.")));
-
-/**
- * Track an event
- * @param eventKey The event name
- * @param userId The user ID associated with the event to track
- * @param eventTags A map of event tag names to event tag values (NSString or NSNumber containing float, double, integer, or boolean)
- */
-- (void)track:(nonnull NSString *)eventKey
```

Comment:
This API should NOT be removed.
we should still be able to send event tags. we just don't pass in the event value as a top level param as a number

Reply:
This is removed due to redeclaration as it is already implemented [here](https://github.com/optimizely/objective-c-sdk/pull/226/files#diff-9bcf60f66151bc781b6adff34a3bc05fR219)